git-cms-merge-topic

```diff
@@ -42,13 +42,9 @@ usage() {
 RED='\033[31m'
 NORMAL='\033[0m'
 
-# get default protocol from git config
-PROTOCOL=$(git config --get cms.protocol)
-HAS_PROTOCOL=$?
-# fallback to default (ignore "mixed")
-if [ "$HAS_PROTOCOL" -ne 0 ] || [ "$PROTOCOL" = "mixed" ]; then
-  PROTOCOL=https
-elif [ "$PROTOCOL" != "https" ] && [ "$PROTOCOL" != "ssh" ] && [ "$PROTOCOL" != "mixed" ]; then
+# get default protocol from git config, or use "mixed" by default
+PROTOCOL=$(git config --get cms.protocol || echo mixed)
```

Comment:
cms-merge-topic doesn't recognize mixed, so the default should be https

Reply:
what will happen if it is set to mixed?
Currently PR tests jobs are failing due to this update and I have set cms.protocol to be `mixed`. Will this fail for cms-merge-topic?

Reply:
The version of the code you're using has this:
```
if [ "$HAS_PROTOCOL" -ne 0 ] || [ "$PROTOCOL" = "mixed" ]; then
  PROTOCOL=https
```

We need to keep the same functionality in this PR.

Reply:
The only check i see is
```
if [ "$PROTOCOL" = "ssh" ]; then
  REPOSITORY=git@github.com:$GITHUB_USER/cmssw.git
else
  REPOSITORY=https://github.com/$GITHUB_USER/cmssw.git
fi
```
so `mixed` should behave just as `https`.

Reply:
Fair point.